parser.go

```diff
@@ -5,6 +5,123 @@ import (
 	"strings"
 )
 
+var tagDecodeSlashMap = map[rune]rune{
+	':':  ';',
+	's':  ' ',
+	'\\': '\\',
```

Comment:
This is so '\\' will decode to '\' (tagDecodeSlashMap is the character that comes *after* a slash.